src/python/src/grpc/_adapter/_c/__init__.pyx

```diff
@@ -0,0 +1,992 @@
+# vim: expandtab:ts=2:sw=2:sts=2
+
+cimport grpc
+cimport cpython
+
+import time
+
+# There are a lot of nested structures in GRPC C core. We wrap all of them in
```

Comment:
Who is the audience for this comment? Is this information that codebase maintainers already familiar with Cython would be expected to know out of hand?